### ISSUE TITLE
Implement Scale Operate?? 

### ISSUE THREAD

**Issue by ly-atdawn** [tensorflow/tensorflow#16116]
------------------------

### System information
- **Have I written custom code: N/A**:
- **OS Platform and Distribution: openSUSE Leap 42.3**:
- **TensorFlow installed from: binary**:
- **TensorFlow version: tensorflow_gpu-1.4.0**:
- **Python version: 2.7.13**: 
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version: 8.0/6.0**:
- **GPU model and memory: 11GB**:
- **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

In Caffe, Scale layer could do this:
top = alpha∗bottom+beta
where bottom is the input, top is the output, alpha and beta are the learnable params.
In Tensorflow, what operate could implement above layer?
Thanks, Help a lot.

**Comment by facaiy** [comment]
Do you mean batch normalization?

**Comment by boluoweifenda** [comment]
I have the same question, the beta (bias) has been implemented as tf.nn.bias_add and it support NCHW or NHWC format, but I didn't find the alpha op. So right now I constuct my own scale layer in python, which seems a little slower.

**Comment by ly-atdawn** [comment]
@facaiy , it is not a batch normalization operate, it just scale the inputs without norm the inputs

**Comment by ly-atdawn** [comment]
@boluoweifenda , I solve this problem like this:
```
with tf.variable_scope("scale", reuse=tf.AUTO_REUSE):
    alpha= tf.get_variable(name='alpha', shape=(channel_num,), trainable=True)
    beta = tf.get_variable(name='beta', shape=(channel_num,), trainable=True)
    output = gamma * input + beta
```
The channel_num is the channel number of feature map. The Scale layer in Caffe, is just operate the channel. I think it is solving the problem.

**Comment by facaiy** [comment]
@ly-atdawn [`tf.layers.dense`](https://www.tensorflow.org/versions/master/api_docs/python/tf/layers/dense)?
> This layer implements the operation: outputs = activation(inputs.kernel + bias)

**Comment by ly-atdawn** [comment]
You can look the Scale layer for detail, it is not same with "tf.layers.dense"........

**Comment by facaiy** [comment]
Oh, the multiplication operation is Hadamard product, not matrix product, right?
http://caffe.berkeleyvision.org/doxygen/classcaffe_1_1ScaleLayer.html#details

**Comment by boluoweifenda** [comment]
@ly-atdawn  Yes, but I find this is even slower than the fused-batchnormalization  (NCHW format)

**Comment by harshini-gadige** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If this is a feature request, please create a new issue by providing all the details asked in this template. Thanks !
